packages/sku/scripts/serve.js

```diff
@@ -33,9 +33,9 @@ const preferredSite = args.site;
 (async () => {
   track.count('serve');
 
-  const targetFolderExists = fs.existsSync(paths.target);
-
-  if (!targetFolderExists) {
+  try {
+    await access(paths.target);
+  } catch {
```

Comment:
Might be useful to have an `exists` util that returns `true`/`false`.

Reply:
Yeah actually having to try/catch everywhere is kinda gross.

Reply:
https://github.com/seek-oss/sku/pull/770/commits/564b44e98abbb575d720132cb8d745f1e0e5f414